cmake/modules/DetectMachine.cmake

```diff
@@ -20,22 +20,40 @@ ENDIF()
 MESSAGE("PROCESSOR: ${CMAKE_SYSTEM_PROCESSOR}")
 SET(LMMS_HOST_X86 FALSE)
 SET(LMMS_HOST_X86_64 FALSE)
+SET(LMMS_HOST_ARM32 FALSE)
+SET(LMMS_HOST_ARM64 FALSE)
+SET(LMMS_HOST_RISCV32 FALSE)
+SET(LMMS_HOST_RISCV64 FALSE)
 
 IF(NOT DEFINED WIN64 AND CMAKE_SIZEOF_VOID_P EQUAL 8)
-    SET(WIN64 ON)
+	# TODO: This seems a bit presumptous
+	SET(WIN64 ON)
 ENDIF()
 
 IF(WIN32)
-	IF(WIN64)
-		SET(IS_X86_64 TRUE)
-		SET(LMMS_BUILD_WIN64 TRUE)
-	ELSE(WIN64)
-		SET(IS_X86 TRUE)
-	ENDIF(WIN64)
-
```

Comment:
Explanation: This block seems to be intended for cross-compilers only (e.g. mingw).  MSVC supports ARM64 targets, we'll set these values manually if on MSVC.  Note, `llvm-mingw` offers ARM64 Windows targets, so it's technically possible that an `mingw` identified compiler can fall-through this use-case.